defaults/api_ents.go

```diff
@@ -0,0 +1,112 @@
+package defaults
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/moby/libentitlement/entitlement"
+	"github.com/moby/libentitlement/secprofile"
+)
+
+const (
+	// APIEntFullID is the API entitlement identifier; the value format is: "api.access:api-id:subset:[allow|deny]"
+	// ex: "api.access:engine.swarm:all:allow"
+	APIEntFullID = "api.access"
+
+	// APIFullControl specifies access control for the whole api
+	APIFullControl = "all"
+)
+
+// Default known APIs and API subsets to control access of
+const (
+	// EngineAPI defines the Moby-Engine API
+	EngineAPI = "engine"
```

Comment:
Instead of `engine` and `swarm`, should this instead be `image`/`container`/`volume`/`swarm`/etc as specified in the Docker API docs? https://docs.docker.com/engine/api/v1.33  I'm not sure I understand the distinction here